src/content/en/tools/workbox/index.md

```diff
@@ -0,0 +1,177 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Service Worker Libraries.
+
+{# wf_published_on: 2015-01-01 #}
+{# wf_updated_on: 2017-05-16 #}
+
+# Workbox {: .page-title }
+
+Use [Workbox](https://workboxjs.org/) to simplify your development by making it
+easy to take advantage of powerful service worker features, eliminate boilerplate code, and automate service worker generation.
+
+<figure class="attempt-right">
+  <img src="/web/tools/workbox/thumb.png" alt="Workbo logo">
+</figure>
+
+Workbox is a collection of loosely-coupled libraries and tools, that focus on
+different service worker features and use-cases. The two core tools that Workbox provides are:
+
+**workbox-sw&mdash;**A service worker library to make managing fetch requests
```

Comment:
A list of definitions are one place it's permissible to use a sentence fragment. Please do so throughout this list. Use parallel sentence constructions as much as possible and start definitions with present tense verbs. 

* A service worker library that makes fetch requests and caching as easy as possible.
*A command-line tool that generates...